integration_test/functions/src/index.ts

```diff
@@ -52,7 +52,9 @@ function callScheduleTrigger(functionName: string, region: string) {
       {
         method: 'POST',
         host: 'cloudscheduler.googleapis.com',
-        path: `projects/${firebaseConfig.projectId}/locations/us-central1/jobs/firebase-schedule-${functionName}-${region}:run`,
+        path: `projects/${
```

Comment:
FYI, this file only contains prettier changes but I decided to bundle this into this PR since they're small.